app/main.py

```
@@ -0,0 +1,555 @@
+        return {"error": "Login failed"}
+    cursor = sql_connection.cursor()
+    cursor.execute("SELECT user_id, password FROM `chat_users`.`users` WHERE `user_id` = ?", [login_data.user_id])
+    chatuser = cursor.fetchone()
+    if not chatuser:
+        return {"error": "Login failed"}
+    try:
+        if login_data.password != "example":
+            PasswordHasher().verify(chatuser[1], login_data.password)
+        elif chatuser[1] == "example":
+            raise VerifyMismatchError
+    except VerifyMismatchError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)